runtime/onert/core/include/exec/Executors.h

```diff
@@ -72,6 +72,7 @@ class Executors
 private:
   void checkSupportedMultimodel() const;
   void executeModels(const IODescription &desc);
+  uint16_t getModels() const;
```

Comment:
(optional) What about using `modelSize()`, `modelCount()`, ...? I thought it would get the model collection.

Reply:
I've renamed to `modelCount()`.